skimage/_shared/meson.build

```diff
@@ -0,0 +1,55 @@
+py3.extension_module('transform',
+  cython_gen.process('transform.pyx'),
+  c_args: cython_c_args,
+  include_directories: [incdir_numpy],
+  install: true,
+  subdir: 'skimage/_shared'
+)
+
+py3.extension_module('fast_exp',
+  cython_gen.process('fast_exp.pyx'),
+  c_args: cython_c_args,
+  include_directories: [incdir_numpy],
+  install: true,
+  subdir: 'skimage/_shared'
+)
+
+py3.extension_module('geometry',
+  cython_gen.process('geometry.pyx'),
+  c_args: cython_c_args,
+  include_directories: [incdir_numpy],
+  install: true,
+  subdir: 'skimage/_shared'
+)
+
+py3.extension_module('interpolation',
+  cython_gen.process('interpolation.pyx'),
+  c_args: cython_c_args,
+  include_directories: [incdir_numpy],
+  install: true,
+  subdir: 'skimage/_shared'
+)
+
+python_sources = [
+  '__init__.py',
+  '_dependency_checks.py',
+  '_geometry.py',
+  '_tempfile.py',
+  '_warnings.py',
+  'coord.py',
+  'filters.py',
+  'lazy.py',
```

Comment:
```suggestion
```
I think the CI fails because it test's on the would-be merged state (what's the proper name?). `lazy.py` was removed in #6577 so it's not found and causes the CI to fail.